<launch>

<rosparam command="load" file="$(find gseg_benchmark)/config/params.yaml" />
<rosparam param="/seq">"00"</rosparam>


<node name="$(anon benchmark)" pkg="gseg_benchmark" type="benchmark" output="screen">
<rosparam param="/algorithm">"patchwork"</rosparam> 
<rosparam param="/seq">"10"</rosparam> 
<rosparam param="/debug">true</rosparam> 
</node>
<node name="rviz" pkg="rviz" type="rviz" args="-d $(find gseg_benchmark)/rviz/ground4r_gpf.rviz"/>


</launch>
